regular-expressions/introduction/basic-syntax-i/regex-wildcards-i.md

```diff
@@ -0,0 +1,87 @@
+---
+author: emmab
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+links:
+- '[The Dot Wildcard](https://www.regular-expressions.info/dot.html){website}'
+- '[Regex metacharacters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes){website}'
+
+---
+# Wildcards I
+
+---
+## Content
+
+Wildcards can be used to add flexibility when you don't want to specify characters. 
+
+> In regex, the `.` wildcard can be used to stand in for nearly *any* character[1] which is why it's the most powerful wildcard.
+
+`/abc.*/`
```

Comment:
this will increase the font-size and apply bold

![image](https://user-images.githubusercontent.com/19802557/82433679-b2d93d80-9a89-11ea-839d-59f367b4a581.png)

```suggestion
### `/abc.*/`
```